mapo/envs/tests/test_navigation.py

```diff
@@ -0,0 +1,152 @@
+# pylint: disable=missing-docstring
+# pylint: disable=protected-access
```

Comment:
This is very nitpicky: you can disable multiple pylint warnings in the same line by separating them with a single comma: `disable=missing-docstring,protected-access`. Don’t worry about it though.

Reply:
Noted!